module/User/src/Mapper/ApiApp.php

```diff
@@ -33,10 +22,10 @@ public function findByAppId($appId)
     }
 
     /**
-     * @return EntityRepository
+     * @inheritDoc
      */
-    public function getRepository()
+    protected function getRepositoryName(): string
     {
-        return $this->em->getRepository(ApiAppModel::class);
+        return 'User\Model\ApiUser';
     }
```

Comment:
Implementation specific; I recommend using the `::class` constant to make obtaining the FQCN easier to understand for the maintainers, as a proper IDE will require a `use` for the entity and link it.

**You are now returning `ApiUser`, but this is the `ApiApp` mapper.**

```php
<?php

namespace User\Mapper;

// (...)
use User\Model\ApiApp as ApiAppModel;

class ApiApp extends BaseMapper
{
    // (...)

    /**
     * @inheritDoc
     */
    protected function getRepositoryName(): string
    {
        return ApiAppModel::class;
    }
}
```

I also recommend we always rename (`as`) the `use` to make it clear which class you are actually using (as we have mappers, models, and services with the same name).